youtube_dl/extractor/libraryofcongress.py

```diff
@@ -0,0 +1,64 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+from .common import InfoExtractor
+
+
+class LibraryOfCongressIE(InfoExtractor):
+    IE_NAME = 'LibraryOfCongress'
+    _VALID_URL = r'https?://(?:www\.)?loc\.gov/item/(?P<id>[0-9]+)'
+    _TESTS = [{
+    'url': 'http://loc.gov/item/90716351/',
+        'info_dict': {
+            'id': '90716351',
+            'ext': 'mp4',
+            'title': 'Pa\'s trip to Mars /'
+        },
+        'params': {
+            # m3u8 download
+            'skip_download': True,
+        }
+    }, {
+        'url': 'https://www.loc.gov/item/97516576/',
+        'only_matching': True,
+    }]
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+
+        self.report_download_webpage(video_id)
+        webpage = self._download_webpage(url, video_id)
+
+        self.report_extraction(video_id)
+        json_id = re.search('media-player-[0-9A-Z]{32}', webpage).group(0).replace('media-player-', '')
+        data = self._parse_json(self._download_webpage('https://media.loc.gov/services/v1/media?id=%s' %json_id, video_id), video_id)
+        data = data.get('mediaObject')
+
+        media_url = data.get('derivatives')[0]['derivativeUrl']
+        media_url = media_url.replace('rtmp', 'https')
```

Comment:
It should work, though when using rtmpdump, it just hangs on 'connecting'.  Would it be better to download the RTMP stream?